### DISCUSSION TITLE
Noindex confusion (warning)

### DISCUSSION THREAD

**Discussion by jollydodo** [thuliteio/doks#1260] [General]
Hey, 

I unfortunately noticed this a little bit too late, but I started with Doks back in September. The frontmatter then had this code in it: ` noindex: true # false (default) or true` (also still visible in the getdoks.org GitHub). However, it seems that this does not do anything. Several pages that shouldn't be indexed were indexed. I looked at the head and found that no meta was inserted. I also can't seem to find any code referring to this bit? 

I then looked at the seo code from Hyas and found that there actually is an implementation and documentation about this. Adding this `robots: "noindex, nofollow"` in my frontmatter works, but it's sad that a lot of pages now need to be changed and are already indexed.

What is the official / supported way of doing this? (and perhaps In the future announce such a big change ;) ) 
After doing this I also disabled the standard robots.txt implementation with a manual simple robots.txt file in the static folder to avoid all traffic by AI crawlers.

**Comment by h-enk** [comment]
Ouch. I'm sorry about this. I should have indeed explicitly communicated this (lesson learned) 😉 

https://seo.gethyas.com/docs/basics/robots/
